pages/your_role/research_software_engineer.md

```diff
@@ -1,6 +1,68 @@
 ---
-title: Research Software Engineer
+title: Research Software Engineer (RSE)
 page_id: research_software_engineer
+description: Software development and quality guidance for Research Software Engineers (RSEs)
+contributors: ["Aleksandra Nenadic"]
 ---
 
-Page about RSEs.
\ No newline at end of file
+
+## Introduction
```

Comment:
Should this be mentioned on this page somewhere - https://direct-framework.github.io/digital-research-competencies-framework/